cmake/common/helpers_common.cmake

```diff
@@ -426,6 +426,14 @@ function(target_export target)
     DESTINATION "${package_destination}"
     COMPONENT Development
     ${exclude_variant})
+
+  if(target STREQUAL libobs)
```

Comment:
The headers include a SIMDE header which make it a dependency of the CMake package and so the finder needs to be installed alongside the package.

Reply:
Yeah the question was more about which ones? Because SIMDE is an implementation detail of `libobs` internals and should not be exposed publicly. It doesn't make sense to expose it from an API point of view and if we do that seems like a mistake to me.

Reply:
`util/sse-intrin.h` used in some provided header.

```
grep -r "sse-intrin.h" /usr/include/obs/. 
/usr/include/obs/./graphics/quat.h:#include "../util/sse-intrin.h"
/usr/include/obs/./graphics/vec3.h:#include "../util/sse-intrin.h"
/usr/include/obs/./graphics/vec4.h:#include "../util/sse-intrin.h"
```

Reply:
Right and of course we inlined all those functions so we created a tight coupling between API users and the internal data types..

That should have never happened, but I guess that's water under the bridge and can be cleaned up once the current API is not used externally anymore in the future.

Reply:
Seems like this thread can be resolved as "todo in a future effort".

Reply:
Ideally it should be fixed before making it an external dependency, because by fixing _that_, we exposed another open wound (namely that it now becomes an explicit public 3rd party dependency) and we just trade one problem (vendored dependency) for another (internal dependency becomes a hard public external dependency) even though there is no architectural reason for that to be the case.

We're doctoring around symptoms again and make a change that is mostly cosmetic instead of fixing the _actual_ issue. But it's a fight I've never won, so I gave up fighting it.